File: test/python/newOrderTest.py
from mexc_sdk.src.mexc_sdk import Spot

client = Spot(api_key="your apiKey", api_secret="your apiSecret")

res = client.new_order_test(symbol="BTCUSDT", side="BUY", order_type="LIMIT", options={ "quantity": 5, "price": 100 })

# if success to post test order to service, res will get {}.
print(res)

File: test/go/newOrderTest.go
package main

import (
	"fmt"

	"mexc-sdk/mexcsdk"
)

func main() {
	apiKey := "your apikey"
	apiSecret := "your apiSecret"
	client := mexcsdk.NewSpot(&apiKey, &apiSecret)

	symbol := "BTCUSDT"
	side := "BUY"
	orderType := "LIMIT"

	options := map[string]interface{}{"quantity": 5, "price": 5}
	res := client.NewOrderTest(&symbol, &side, &orderType, &options)
	fmt.Println(res)
}